packages/app/cypress/integration/components/search/SearchPage.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 Spotify AB
```

Comment:
```suggestion
 * Copyright 2021 Spotify AB
```

goes for all notice headers